README.md

```diff
@@ -1,5 +1,5 @@
-## A Dockerfile for the [ArchiveTeam Warrior](https://www.archiveteam.org/index.php?title=ArchiveTeam_Warrior)
 
+## A Dockerfile for the [ArchiveTeam Warrior](https://www.archiveteam.org/index.php?title=ArchiveTeam_Warrior)
```

Comment:
Is the line shifted to a different line a typo?

Reply:
whops. yes.